torch/csrc/CudaIPCTypes.cpp

```diff
@@ -32,6 +32,7 @@ struct CudaIPCGlobalEntities {
   std::shared_ptr<CudaIPCRefCountersFile> next_available_ref_counters_file_;
   CudaIPCSentDataLimbo CudaIPCSentDataLimbo_;
   // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
+  // NOLINTNEXTLINE(modernize-use-equals-default)
```

Comment:
```suggestion
  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init,modernize-use-equals-default)
```
You need to modify annotation script to merge two NOLINTNEXTLINE statements together